jep/1/README.adoc

```diff
@@ -0,0 +1,939 @@
+can be conducted via the GitHub JEP Repository footnoteref:[repo] and pull
+requests.
+
+==== Conributor
+
+A JEP may have any number of "Contributors" who
+help write, implement, discuss, or offer feedback about that JEP.
+One contributor might do only one or any combination of these
+of these things during any part of the life of a JEP.
+
+==== Sponsor
+
+Each JEP has at least one "Sponsor".
+
+The JEP Sponsor is a contributor who is responsible for the JEP overall -
+building consensus for that JEP within the community,
+documenting dissenting opinions, coordinating contributors work,
+and generally ensuring the JEP meets the style, format,
+and quality guidelines described below.
+A Sponsor is also responsible for maintaining the JEP after it is finalized.
+
+A JEP may have more than one Sponsor, especially after it has been finalized
+and is being maintained over time.
+For simplicity, this document uses the singular
+("JEP Sponsor", "sponsor")
+when referring the one or more people in the role of "Sponsor" of a JEP.
+
+[[requirement-levels]]
+==== Must/Should/May
+
+JEP documents _must_ follow link:https://tools.ietf.org/html/rfc2119[RFC 2119]
+which defines key words to "indicate requirement levels". These are listed
+below:
+
+. **MUST** This word, or the terms "REQUIRED" or "SHALL", mean that the
+   definition is an absolute requirement of the specification.
+. **MUST NOT** This phrase, or the phrase "SHALL NOT", mean that the
+   definition is an absolute prohibition of the specification.
+. **SHOULD** This word, or the adjective "RECOMMENDED", mean that there
+   may exist valid reasons in particular circumstances to ignore a
+   particular item, but the full implications must be understood and
+   carefully weighed before choosing a different course.
+. **SHOULD NOT** This phrase, or the phrase "NOT RECOMMENDED" mean that
+   there may exist valid reasons in particular circumstances when the
+   particular behavior is acceptable or even useful, but the full
+   implications should be understood and the case carefully weighed
+   before implementing any behavior described with this label.
+. **MAY** This word, or the adjective "OPTIONAL", mean that an item is
+   truly optional.
+
+NOTE: When choosing to go counter to SHOULD or SHOULD NOT guidance,
+the reasons behind that choice SHOULD be documented.
+
+
+=== JEP Workflow
+
+==== Overview
+
+Before delving into the details of the JEP workflow,
+let's take a high-level look at how JEP might go.
+
+. **<<start, Inital Discussion>>** - Andrea has an idea for new feature emails jenkinsci-dev@googlegroups.com.
+  She discusses the idea with the group, determining that the idea is worth pursuing.
+  She chooses to be the "<<Sponsor>>" for this potential JEP.
+  She <<discussion, gathers initial feedback>> from the community, adjusts her design as needed,
+  records the reasons for design choices, and keeps track of differing views.
```

Comment:
Nowhere is it explicitly stated that someone other than the person proposing something could be its sponsor.

Reply:
Updated the Sponsor section to add this. 
RESOLVED.